sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs

```diff
@@ -207,6 +207,34 @@ public ServiceBusAdministrationClient(string connectionString)
         ///
         /// <remarks>Works with any claim (Send/Listen/Manage).</remarks>
         /// <returns><see cref="NamespaceProperties"/> containing namespace information.</returns>
+        ///
+        /// <exception cref="UnauthorizedAccessException">No sufficient permission to perform this operation.
```

Comment:
```suggestion
        /// <exception cref="UnauthorizedAccessException">Insufficient permission to perform this operation.
```

This read a bit awkward to me, as written.  If we decide to tweak this, we should include the other instances that follow.